main.go

```diff
@@ -163,24 +167,28 @@ func delete(cmd *cobra.Command, args []string) error {
 }
 
 func listDbs(cmd *cobra.Command, args []string) error {
+	list, err := getDbs()
+	for _, db := range list {
+		fmt.Println("@" + db.Name())
+	}
+	return err
+}
+
+// TODO: write test for this
+func getDbs() ([]os.DirEntry, error) {
 	cc, err := client.NewClientWithDefaults()
 	if err != nil {
-		return err
+		return nil, err
 	}
 	dd, err := cc.DataPath()
 	if err != nil {
-		return err
+		return nil, err
 	}
 	dbs, err := os.ReadDir(filepath.Join(dd, "kv"))
 	if err != nil {
-		return err
+		return nil, err
 	}
-	for _, d := range dbs {
-		if d.IsDir() {
-			fmt.Println("@" + d.Name())
-		}
-	}
-	return nil
+	return dbs, nil
```

Comment:
We're losing this information, not checking for `d.IsDir()`, after returning dir entries.
```suggestion
       rv := make([]os.DirEntry, 0)
       for _, d := range dbs {
		if d.IsDir() {
			rv = append(rv, d)
		}
	}
	return rv, nil
```

Reply:
Or even better, make `getDbs()` returns a list of db names as strings prepended with `@` 
`func getDbs() ([]string, error)`